app.py

```diff
@@ -23,6 +24,14 @@ def create_app(config_name):
             graphiql=True   # for having the GraphiQL interface
         )
     )
+    app.add_url_rule(
+        '/_healthcheck',
+        view_func=GraphQLView.as_view(
+            '_healthcheck',
+            schema=healthcheck_schema,
+            graphiql=True   # for healthchecks
+        )
+    )
```

Comment:
I thought the health check url would be a template for visualization if the services are okay but not an endpoint. What I mean by this, is having a route for a view function that renders this template.

Reply:
@PatrickCmd ,the story description was changed to an endpoint.